docs/ref/forms/api.txt

```diff
@@ -1210,24 +1210,33 @@ Methods of ``BoundField``
 
 .. method:: BoundField.label_tag(contents=None, attrs=None, label_suffix=None)
 
+    Renders the label tag of a form field with the template specified by
+    :attr:`.Form.template_name_label`.
+
+    The available context is:
+
+    * ``field``: The bound field
```

Comment:
ah but is it the bound _Field_, or the _BoundField_? The nomenclature unfortunately requires knowing there's a difference to begin with... Contextually one may _assume_ it's `this BoundField instance` but maybe we can spell it out better?

Reply:
Yes, good idea. Also in the template this leads to `field.field. ...`. It took me a while to work out what `field` actually was when working learning crispy-forms. I'll add a `tip` to try and explain, but happy for it to be taken out if folk think it's too much.